config/checkstyle/checkstyle.xml

```diff
@@ -163,7 +163,7 @@
         <module name="FinalClass"/>
         <!--<module name="HideUtilityClassConstructor"/>-->
         <module name="InterfaceIsType"/>
-        <module name="VisibilityModifier"/>
+<!--        <module name="VisibilityModifier"/>-->
```

Comment:
Tohle bych asi nevypinal.

Reply:
Nebo aspon globalne.

Reply:
Toto nedovoluje udelat protected promennou a pak s ni pracovat ve tridach, ktere ji rozsiruji, coz nedava moc smysl. Ale klidne to tam necham, ale pak budu muset predelat volani v tech output printrech vzdycky na `getOutput.println()` namisto `output.println()`, coz mi nedava moc smysl.

Reply:
No já bych byl klidně pro to mít to private a třeba protected getter.
Nebo bych to Checkstyle vypnul jen pro tuhle třídu. Já se radši saháním na atributy ostatních tříd, i když třeba rodičovským radši vyhýbám. Nechám na tobě.